### PR TITLE
ignore error if pod not found

### PR DESCRIPTION
**Pull request by dv-blk**
##### Purpose
Error delete pod (such as pod not found) stops execution

##### Notes
I'll update our submodules after this is merged

### CODE REVIEW

File: deploy_to_kubernetes_cron.sh [hunk 1]
```diff
@@ -29,7 +29,7 @@ for command in "${!commands[@]}"; do
   echo SCHEDULE "$SCHEDULE"
   echo PRIMARY_IMAGE "$PRIMARY_IMAGE"
 
-  kubectl delete cronjob $NAME
+  kubectl delete cronjob $NAME || true
```
Comment (jakeorr) [inline deploy_to_kubernetes_cron.sh]: I think it would be better to conditionally execute this instead of executing it with a bogus argument.

Comment (dv-blk) [reply deploy_to_kubernetes_cron.sh]: why?

**Comment by dv-blk** [comment]
test

File: deploy_to_kubernetes_cron.sh (hunk 1 above)
Comment (jakeorr) [reply deploy_to_kubernetes_cron.sh]: talked offline, this is somewhat of a standard bash thing. the output of the command will go to the pipe true and cause the script to get past the line if it errors. sounds good to me. 👍

**Review by jakeorr — APPROVED** [review]

File: deploy_to_kubernetes_cron.sh [hunk 2]
```diff
@@ -29,7 +29,8 @@ for command in "${!commands[@]}"; do
   echo SCHEDULE "$SCHEDULE"
   echo PRIMARY_IMAGE "$PRIMARY_IMAGE"
 
-  kubectl delete cronjob $NAME
+  # Ignore exit code with pipe operator
+  kubectl delete cronjob $NAME || true
```
Comment (ryancampbell) [inline deploy_to_kubernetes_cron.sh:33]: FYI you can use --ignore-not-found=false to ignore "not found" errors

**Pull request merged**